zerver/webhooks/beanstalk/view.py

```diff
@@ -26,10 +25,10 @@ def _wrapped_view_func(request: HttpRequest, *args: Any, **kwargs: Any) -> HttpR
         try:
             auth_type, encoded_value = request.META['HTTP_AUTHORIZATION'].split()  # type: str, str
             if auth_type.lower() == "basic":
-                email, api_key = base64.b64decode(force_bytes(encoded_value)).decode('utf-8').split(":")
+                email, api_key = base64.b64decode(encoded_value).decode('utf-8').split(":")
```

Comment:
Similar to the commit https://github.com/zulip/zulip/commit/da2744812fbc41c94249cfa78907b04584168c96